### ISSUE TITLE
Remove INSIDE_ENCLAVE-gated blocks from PBFT

### ISSUE THREAD

**Issue by achamayou** [microsoft/CCF#805]
There are a number of sections of code in REplica.cpp that are gated by INSIDE_ENCLAVE because they apparently write to disk. That's neither something we want in enclave, nor outside, disk I/O if any is needed should be separate and pluggable.

@ashamis because the code still works currently, I assume they are in fact not needed for any purpose other than maybe the old standalone tests, so we can probably just remove that code when those are removed.

**Comment by ashamis** [comment]
This is rements of old code, mainly "proactive recovery". I will delete this in the near future
